flepimop/gempyor_pkg/src/gempyor/model_info.py

```diff
@@ -76,9 +76,8 @@ def __init__(
         # Auto-detect old config
         if config["interventions"].exists():
             raise ValueError(
-                """This config has an intervention section, and has been written for a previous version of flepiMoP/COVIDScenarioPipeline \
-                             Please use flepiMoP Version 1.1 (Commit SHA: 0c30c23937dd496d33c2b9fa7c6edb198ad80dac) to run this config. \
-                             (use git checkout v1.1 inside the flepiMoP directory)"""
+                "This config has an intervention section, which is only compatible with a previous version of flepiMoP/COVIDScenarioPipeline. "
```

Comment:
This is 143 chars, can we bring this down to 92 or less?

Reply:
Oh weird I actually think I missed this one somehow in my initial sweep. Thanks for pointing out.